backend/src/main/java/quarano/actions/ActionItemRepository.java

```diff
@@ -37,4 +37,9 @@ default ActionItems findUnresolvedByCase(TrackedCase trackedCase) {
 			+ " where i.slot = :slot" //
 			+ " and i.personIdentifier = :personIdentifier")
 	ActionItems findDiaryEntryMissingActionItemsFor(TrackedPersonIdentifier personIdentifier, Slot slot);
+
+	@Query("select i from QuarantineEndingActionItem i" //
+			+ " where i.personIdentifier = :personIdentifier")
+	ActionItems findQuarantineEndingActionItemsFor(TrackedPersonIdentifier personIdentifier);
```

Comment:
Do we need unit tests for new queries? I didn't find any for `findDiaryEntryMissingActionItemsFor`... so I left them out. 🤭